src/prefect/config.toml

```diff
@@ -56,6 +56,7 @@ endpoint = "https://api.prefect.io"
 graphql = "${cloud.api}/graphql"
 use_local_secrets = true
 heartbeat_interval = 30.0
+heartbeat_mode = "process" # ['process', 'thread', 'off']
```

Comment:
If a user toggles a heartbeat to 'off' isn't it just going to cause all their long running flows to be killed? I guess they could toggle heartbeats _and_ the checking service. Sounds like a complicated toggle for them but 🤷

Reply:
I think in general this config is a power user config -- if we hear back positive results from our affected users, the idea would be to migrate entirely to threads